### PR TITLE
Stock api

### PR DESCRIPTION
**Pull request by Arjun-sna**
API to update stock availability of groceries.

### CODE REVIEW

File: packages/CB-serverless-backend/api/groceries/stock.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+	dataToUpdate.forEach(rowData => {
+		const params = {
+			TableName : 'grocery',
+			Key: {
+				groceryId: parseInt(rowData.groceryId),
+			},
+			UpdateExpression: `set availableQty=:updatedQty`,
+			ExpressionAttributeValues:{
+				":updatedQty":rowData.availableQty,
```
Comment (prasanna1211) [inline packages/CB-serverless-backend/api/groceries/stock.js]: Should we add with the existing stock or just fix the stock as this ?

File: packages/CB-serverless-backend/api/groceries/stock.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+import awsConfigUpdate from '../../utils/awsConfigUpdate';
+import getErrorResponse from '../../utils/getErrorResponse';
+import getSuccessResponse from '../../utils/getSuccessResponse';
+
+awsConfigUpdate();
+
+export const updateStock = async (event, context, callback) => {
+  context.callbackWaitsForEmptyEventLoop = false;
+	const dataToUpdate = JSON.parse(event.body);
```
Comment (prasanna1211) [inline packages/CB-serverless-backend/api/groceries/stock.js]: We need to handle errors for 400. Consider Empty body or id not present may be ?

File: packages/CB-serverless-backend/api/groceries/stock.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+				":updatedQty":rowData.availableQty,
+			},
+			ReturnValues: "UPDATED_NEW"
+		};
+		promiseArray.push(documentClient.update(params).promise())
+	});
+
+	Promise.all(promiseArray)
+		.then(data => callback(null, getSuccessResponse({success: true})))
+		.catch(error => getErrorResponse(callback, 500, error))
```
Comment (prasanna1211) [inline packages/CB-serverless-backend/api/groceries/stock.js]: I guess improper data will be thrown here. So no need to check it explicitly.

File: packages/CB-serverless-backend/api/groceries/stock.js (hunk 2 above)
Comment (Arjun-sna) [reply packages/CB-serverless-backend/api/groceries/stock.js]: the client should always send the updated value. Like if the current value is 12 and if it wants to add 2 more to stock, then the client should send 14 as value. If not there will be unnecessary read for every update.

File: packages/CB-serverless-backend/api/groceries/stock.js (hunk 1 above)
Comment (Arjun-sna) [reply packages/CB-serverless-backend/api/groceries/stock.js]: the client should always send the updated value. Like if the current value is 12 and if it wants to add 2 more to stock, then the client should send 14 as value. If not there will be unnecessary read for every update.

**Pull request merged**